aries-site/src/layouts/main/Layout.js

```diff
@@ -45,34 +45,36 @@ export const Layout = ({ children, title }) => {
                 content="width=device-width, initial-scale=1.0"
               />
             </Head>
-            <Nav title="Aries" background="background-subtle">
-              <AnchorGroup
-                items={[
-                  { label: 'Start', href: '/start' },
-                  { label: 'Foundation', href: '/foundation' },
-                  { label: 'Design', href: '/design' },
-                  { label: 'Develop', href: '/develop' },
-                  { label: 'Resources', href: '/resources' },
-                ]}
-              />
-            </Nav>
-            <Box
-              direction="row"
-              // pad uses Nav pad from aries-core
-              pad={{
-                horizontal: size !== 'small' ? 'xlarge' : 'large',
-                vertical: 'large',
-              }}
-            >
-              {size !== 'small' && (
-                <Box fill="vertical">
-                  <SideBar
-                    items={SideBarItemList[selectedNav]}
-                    prefix={selectedNav}
-                  />
-                </Box>
-              )}
-              <Box flex>{mainContent[0]}</Box>
+            <Box width={{ max: 'xlarge' }} margin="auto">
```

Comment:
why do you need the extra Box? Nav should be able to accept both `width={{ max: 'xlarge' }} margin="auto"`?

Reply:
In this case, I was just wrapping the entire app in a bounding box, but I agree that Nav should also be able to take those props as well. Let's put a hold on this PR though, because I am waiting on a clarification from Chris is the nav background should extend the full width of the window even if the content only spans width of xlarge.